.github/workflows/push_pr.yml

```diff
@@ -11,6 +11,7 @@ on:
       - README.md
       - 'docs/**'
       - 'examples/**'
+  pull_request:
```

Comment:
we need to discuss this a bit further as if the secrets we are using in the repo are not present in the fork it will not work.

Reply:
You are totally right! We might need to separate testing workflows with the ones that need secrets (login) like snyk. 

Thanks for pointing that out